linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -48,6 +49,11 @@ pub struct SharedLocalKubernetesNetTestingConfig {
     pub binaries: Option<Option<PathBuf>>,
 }
 
+struct BuildConfig {
+    binaries: Option<Option<PathBuf>>,
```

Comment:
nit: Typically I would create an enum for this like `enum { Case1, Case2, Case3(PathBuf) }`.